### DISCUSSION TITLE
The first run reports an error

### DISCUSSION THREAD

**Discussion by Beipy** [hoppscotch/hoppscotch#1674] [Q&A] [answered]
Project `npm run dev` reports an error when it runs for the first time.
How to solve the error message? 
thank you  Looking forward to reply

<img width="1143" alt="iShot2021-05-28 12 01 34" src="https://user-images.githubusercontent.com/33574750/119927680-7699c080-bfac-11eb-8719-174a96f4e818.png">

**Comment by liyasthomas** [comment, marked as answer]
In readme, there's a [Step 0: Settings up environment variables for local development setup](https://github.com/hoppscotch/hoppscotch#developing).

Rename `.env.example` file in root directory to `.env`. Optionally add your own Firebase config keys to `.env` file.

**Reply by Beipy** [reply]
thank you
